pyensembl/genome.py

```diff
@@ -438,7 +442,11 @@ def transcript_sequence(self, transcript_id):
         if self.transcript_sequences is None:
             raise ValueError(
                 "No transcript FASTA supplied to this Genome: %s" % self)
-        return self.transcript_sequences.get(transcript_id)
+        sequence = [x for x in map(lambda x: x.get(transcript_id), self.transcript_sequences) if x is not None]
```

Comment:
It might make more sense to combine the transcript sources when we're constructing `transcript_sequences` and check for duplicates then. 

Also, style-wise, the double usage of `x` made this particularly tricky to parse.